### ISSUE TITLE
No longer suggests the keywords of the Libraries

### ISSUE THREAD

**Issue by mayribeirofernandes** [gliviu/autocomplete-robot-framework#11]
Hello!

My _autocomplete-robot-framework_ plugin stopped working a few days ago, I didn't update anything and I didn't even update my ATOM. The plugin is working in parts, it is only suggesting keywords for my **resources** and no longer suggests the keywords of the **libraries** declared in the _settings_ of my file. My Robot Framework course students have been reporting the same problem with them for a few weeks. Are you aware of anything that is preventing the plugin from working completely?

My ATOM: 1.48.0
My Operating System: Windows 10 Pro
My plugin: 3.8.0

**Comment by gliviu** [comment]
I'll take a look and let you know.

**Comment by gliviu** [comment]
Looks like new robotframework release 3.2 breaks this package.
Until I investigate you can downgrade.
```bash
pip uninstall robotframework
pip install 'robotframework==3.1.2'
```

**Comment by mayribeirofernandes** [comment]
That's right, I returned the robotframework to 3.1.2 and the KWs were suggested again! I'll be waiting for compatibility with 3.2! Thank you!

**Comment by gliviu** [comment]
Meanwhile I published 3.9.0 that should fix the issue with robotframework 3.2. If you see any problems please let me know.

**Comment by mayribeirofernandes** [comment]
Perfect!! It's working again, thank you very much, anything I report here again.
